packages/vector_graphics_codec/test/vector_graphics_codec_test.dart

```diff
@@ -633,6 +633,31 @@ void main() {
       OnDrawText(textId, paintId),
     ]);
   });
+
+  test('Encodes image data', () {
```

Comment:
consider having some test(s) that encode images along with other things, so that we get coverage of making any mistakes of over/under bumping the buffer pointer.

Reply:
Done